README.md

```diff
@@ -19,38 +19,40 @@ To build Shotover from source please refer to [the contributing documentation](h
 
 Shotover is a high performance, configurable and extensible L7 data-layer proxy for controlling, managing and modifying the flow of database requests in transit. It can be used to solve many different operational and interoperability challenges by transparently intercepting and transforming queries. It is transparent in the sense that it can be plugged into your architecture without requiring application change.
 
-Shotover currently supports intercepting requests for the following technologies (sources):
+Shotover currently supports the following databases:
 
-* Cassandra (CQL4)
-* Redis (RESP2)
+* Cassandra (CQL4 + CQL5)
+* Valkey/Redis (RESP2)
 * Kafka (Kafka Wire Protocol)
 
-It currently supports writing output to the following technologies (sinks):
-
-* Cassandra
-* Redis
-* Kafka
-
 ## What problems does Shotover solve?
 
 Concrete examples where Shotover has been applied include:
 
-* [Multi-region, active-active redis](https://github.com/shotover/shotover-examples/tree/main/redis-backup-cluster)
-<!--* [Cassandra query caching in redis, with a query audit trail sent to kafka](shotover-proxy/tests/test-configs/cass-redis-kafka/)-->
+* [Multi-region, active-active valkey](https://github.com/shotover/shotover-examples/tree/main/valkey-backup-cluster)
+* [Cassandra AWS PrivateLink](https://github.com/shotover/shotover-examples/blob/main/cassandra-1-many/topology.yaml)
+* [Kafka GCP Private Service Connect](https://github.com/shotover/shotover-examples/blob/main/kafka-1-many/topology.yaml)
 
-More broadly, Shotover is designed to be used for a very wide ranging class of problems where it is useful to transparently intercept a database call and redirect it. This allows you to change the behaviour of running applications at the infrastructure level without change to the application code itself.
+More broadly, Shotover is designed to be used for a very wide ranging class of problems where it is useful to transparently intercept a database call and redirect/rewrite it. This allows you to change the behaviour of running applications at the infrastructure level without change to the application code itself.
 Some examples where we envisage Shotover could be deployed include:
 
+* Enabling use of AWS private link or GCP private service connect for database's that do not support distinct load balancer nodes.
```

Comment:
typo in `database's`, `private link` -> `PrivateLink` and `private service connect` -> `Private Service Connect`?

Reply:
fixed